features/registry/is.feature

```diff
@@ -109,6 +109,7 @@ Feature: Testing imagestream
     When I run the :new_app_as_dc client command with:
       | app_repo | quay.io/openshifttest/ruby-27:multiarch~https://github.com/sclorg/ruby-ex.git |
     Then the step should succeed
+    And the "ruby-27:multiarch" image stream tag was created
```

Comment:
How about directly stated that `I wait for the "xxx_name_xxx" xxx_type_xxx to appear` ? There is no step update required in step_definitions in this way.

If you need to specify how long to wait, you can use `I wait for the "xxx_name_xxx" xxx_type_xxx to appear up to xxx seconds`
If you also need to specify the project, you can use `I wait for the "xxx_name_xxx" xxx_type_xxx to appear in the "xxx" project up to xxx seconds`

Reply:
@liangxia It works, updated, thanks CC @zhouying7780